static/js/components/VideoPlayer.js

```diff
@@ -285,27 +286,24 @@ class VideoPlayer extends React.Component<*, void> {
     })
   }
 
-  selectPlaylist = () => {
-    const sortByBandwidth = R.sortBy(R.path(["attributes", "BANDWIDTH"]))
-    const playlists = sortByBandwidth(
-      this.player.tech_.hls.playlists.master.playlists
-    )
+  selectPlaylist = (opts: { defaultPlaylistSelector: () => any }) => {
+    const { defaultPlaylistSelector } = opts
     // Always start with highest bandwidth for first 10 seconds
     if (this.player.tech_.currentTime() < 10) {
+      const sortByBandwidth = R.sortBy(R.path(["attributes", "BANDWIDTH"]))
+      const playlists = sortByBandwidth(
+        this.player.tech_.hls.playlists.master.playlists
+      )
       return _.last(playlists)
     }
-    // Return playlist with highest bandwidth <= system bandwidth
```

Comment:
This overrides the native HLS algorithm so that the video player's size in the browser is ignored; and should still be used unless someone manually selects a quality level.